docs/components/rule.md

```diff
@@ -96,6 +96,9 @@ labels:
   [ <labelname>: <labelvalue> ]
 ```
 
+Note: If you make use of recording rules, make sure that you expose your Ruler instance as a store in the Thanos Querier so that the new time series can be queried
+as part of Thanos Query. You can do this by adding a new `--store <thanos-ruler-ip>` command-line argument to the Thanos Query command.
```

Comment:
Maybe worth mentioning that it is not the only way of doing this:
```suggestion
as part of Thanos Query. One of the ways you can do this is by adding a new `--store <thanos-ruler-ip>` command-line argument to the Thanos Query command.
```

Reply:
Thanks, that makes sense. As I have been banging my head for days without this 'issue'. I am wondering what other ways would be? As you referring to using a service discovery url or something totally different?